Pizzeria/src/main/java/com/pizzeria/training/controllers/OrderController.java

```diff
@@ -62,15 +70,33 @@ public Order updateOrder(@RequestParam ObjectId _id, @RequestBody Order updateOr
 		return orderServ.save(updateOrder);
 	}
 	
+	@PutMapping("/completeOrder")
+	public ResponseEntity<String> completeOrder(@RequestBody String orderId) {
+		return orderServ.updateStatus(new ObjectId(orderId));
+	}
+	
+	@GetMapping("/getOrdersByStatus/{orderStatus}")
```

Comment:
Doesn't seem to match with the HTML standards of endpoints being nouns

Reply:
I would recommend putting the order status as an optional path variable or request param on the existing get /orders endpoint (see lines 55-58).